homework-submission/week1-react/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import todoLogo from './todo.png';
+import './App.css';
+import {inject, observer} from 'mobx-react';
+import Todo from "./component/todo.js";
+import Add from "./component/add";
+import Update from "./component/update";
+
+@inject('myTodoApp')
+@observer
+
+class App extends Component {
+
+  render() {
+    const {myTodoApp} = this.props;
+
+    const isEmpty = myTodoApp.data.todos.length < 1 || myTodoApp.data.todos === undefined;
```

Comment:
This may be shortened a bit for a code styling if you would like 🙂:
```suggestion
    const isEmpty = !myTodoApp.data.todos.length || !myTodoApp.data.todos;
```
Or may be :
```suggestion
    const isEmpty = !(myTodoApp.data.todos.length || myTodoApp.data.todos);
```

Reply:
Thank you for taking my attention to `truthy falsy` . actually I know what it is but I was not using them directly. it looks simple